Description: The Debian location of the Grunt modules isn't found automatically
 yet. So we need to help Grunt.
 .
 Actually, as of Grunt 1.0.1-7, this is possible, but while Grunt isn't
 backported to stretch, let's keep the patch.
 .
 Also not all modules are packaged yet, so disable cssmin for now.
Author: Paul Gevers <elbrus@debian.org>
Forwarded: not-needed
Index: jquery-tablesorter/Gruntfile.js
===================================================================
--- jquery-tablesorter.orig/Gruntfile.js
+++ jquery-tablesorter/Gruntfile.js
@@ -373,16 +373,16 @@ module.exports = function( grunt ) {
 
 	});
 
-	grunt.loadNpmTasks( 'grunt-contrib-clean' );
-	grunt.loadNpmTasks( 'grunt-contrib-jshint' );
-	grunt.loadNpmTasks( 'grunt-htmlhint' );
-	grunt.loadNpmTasks( 'grunt-contrib-qunit' );
-	grunt.loadNpmTasks( 'grunt-contrib-concat' );
-	grunt.loadNpmTasks( 'grunt-contrib-uglify' );
-	grunt.loadNpmTasks( 'grunt-contrib-copy' );
-	grunt.loadNpmTasks( 'grunt-contrib-watch' );
-	grunt.loadNpmTasks( 'grunt-contrib-cssmin' );
-	grunt.loadNpmTasks( 'grunt-jscs' );
+	grunt.loadNpmTasks( 'grunt-contrib-clean', '/usr/lib/nodejs' );
+	grunt.loadNpmTasks( 'grunt-contrib-jshint', '/usr/lib/nodejs' );
+	grunt.loadNpmTasks( 'grunt-htmlhint', '/usr/lib/nodejs' );
+	grunt.loadNpmTasks( 'grunt-contrib-qunit', '/usr/lib/nodejs' );
+	grunt.loadNpmTasks( 'grunt-contrib-concat','/usr/lib/nodejs' );
+	grunt.loadNpmTasks( 'grunt-contrib-uglify','/usr/lib/nodejs' );
+	grunt.loadNpmTasks( 'grunt-contrib-copy','/usr/lib/nodejs' );
+	grunt.loadNpmTasks( 'grunt-contrib-watch', '/usr/lib/nodejs' );
+	grunt.loadNpmTasks( 'grunt-contrib-cssmin', '/usr/lib/nodejs' );
+	grunt.loadNpmTasks( 'grunt-jscs', '/usr/lib/nodejs' );
 
 	grunt.registerTask( 'test', [ 'jscs', 'jshint', 'htmlhint', 'qunit' ] );
 
@@ -391,8 +391,8 @@ module.exports = function( grunt ) {
 		'copy',
 		'concat',
 		'uglify',
-		'cssmin',
-		'clean:css',
+//		'cssmin',
+//		'clean:css',
 		'updateManifest'
 	];
 
